frontend/pages/fishbowl/thankyou/[fid].tsx

```diff
@@ -14,7 +14,7 @@ import dynamic from 'next/dynamic';
 import useTranslation from 'next-translate/useTranslation';
 
 import { ROUTE_FISHBOWL_CREATE, ROUTE_NOT_FOUND } from 'app.config';
-import { dataLayerPush } from 'lib/analytics';
+import { dataLayerPush, pushEventDataLayer } from 'lib/analytics';
```

Comment:
I would consider renaming `dataLayerPush` function. In my opinion it is confusing as they both push events to Analytics

Reply:
Yeah, maybe we should use something like `pageView...`. Should we fix this in the next PR?